addons/payment_custom/views/payment_provider_views.xml

```diff
@@ -14,6 +24,19 @@
                     {'invisible': [('code', '=', 'custom')]}
                 </attribute>
             </group>
+            <field name="pending_msg" position="after">
+                <div class="o_row replace_pending_msg" colspan="2"
+                     attrs="{'invisible': ['|', ('code', '!=', 'custom'), ('custom_mode', '!=', 'wire_transfer')]}">
+                    <button class="fa fa-refresh fa-fw btn-link"
+                            data-tooltip-touch-tap-to-show="true"
+                            data-tooltip-template="web.FieldTooltip"
+                            data-tooltip-info="{&quot;field&quot;:{&quot;help&quot;:&quot;Reset pending message&quot;}}"
```

Comment:
This line is kind of not human friendly... I didn't try it, but you could look in the `hr_contrat_views.xml`, this data-tooltip-info is also used. Though, as it's exactly the same message as in the title, I'm wondering if this tooltip is really useful... :thinking:

Reply:
I just removed it :)